src/models/CountryBrowserVersionHits.js

```diff
@@ -0,0 +1,52 @@
+const Joi = require('joi');
+const BaseModel = require('./BaseModel');
+
+const schema = {
+	countryIso: Joi.string().length(2).required(),
+	browserVersionId: Joi.string().length(2).required().allow(null),
```

Comment:
This looks wrong.

Reply:
Needed to pass validation, as long as `browserVersionId` we will get through SQL

Reply:
Oh your were talking about `length`... yeah, copy-pasta failed me